tests/integration/dotnet/basic/Program.cs

```diff
@@ -129,7 +128,7 @@ static Task<int> Main(string[] args)
                 RoleRef = new RoleRefArgs
                 {
                     Kind = "ClusterRole",
```

Comment:
You might need to change this to `Role` to get CI passing. I'm not sure which role is needed for the CI user to do this, but it's not working. I tested it with my `Owner` account and it worked.